README.asc

```diff
@@ -39,7 +39,7 @@ be sitting in front of a prompt:
 
 ----
 Connected to Test Cluster at localhost:9160.
-[cqlsh 6.0.0 | Cassandra 4.0.2 | CQL spec 3.4.5 | Native protocol v5]
```

Comment:
Mmmm this should be `Cassandra 4.0.2` -> `Cassandra 4.1`. I can do that on commit though.